locales/fr.schema.json

```diff
@@ -2515,7 +2515,7 @@
                 "label": "Cercle"
               },
               "options__2": {
-                "label": "Carré"
+                "label": "Square"
```

Comment:
Here as well. It's not consistent through that file. Sometimes it's good sometimes it isn't/